//package supermarket.protocol;
//·¢ËÍĞÄÌø

message MceHeartbeat {
    optional bool update = 1[default = false];
}
